addons/mrp/models/mrp_unbuild.py

```diff
@@ -132,6 +132,28 @@ def _unlink_except_done(self):
         if 'done' in self.mapped('state'):
             raise UserError(_("You cannot delete an unbuild order if the state is 'Done'."))
 
+    def _create_finished_move_line(self, finished_move):
+        return self.env['stock.move.line'].create({
+            'move_id': finished_move.id,
+            'lot_id': self.lot_id.id,
+            'qty_done': finished_move.product_uom_qty,
```

Comment:
I can see that you've changed `self.product_qty` to `finished_move.product_uom_qty`, was this intentional?

Reply:
That was my mistake. It should be `self.product_qty`. I fixed it.